### PR TITLE
Afd 22 book details view desktop

### PR DESCRIPTION
**Pull request by TheGuardingDark**
Added breakpoints for desktop view of details page

### CODE REVIEW

File: src/assets/styles/scss/details.scss [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+.bodyWrapper {
+    min-width: fit-content;
+}
+.pageWrapper {
+    grid-template-columns: 1fr .5fr 1fr;
+    grid-template-rows: 2fr .5fr;
+    padding: 2rem 6rem 4rem 5rem;
+}
+.nav {
+    .nav {
```
Comment (aperezsantos) [inline src/assets/styles/scss/details.scss:17]: Just a quick thought: are these supposed to be two different classes?

File: src/assets/styles/scss/details.scss [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+}
+.book {
+    grid-column: 2/4;
+    grid-row: 1;
+    text-align: left;
+    padding: 1rem 8rem 0 4rem;
+    grid-template-rows: repeat(4, .5fr) 2fr;
+    &__text {
+        font-size: 1rem;
+        &--author {
```
Comment (aperezsantos) [inline src/assets/styles/scss/details.scss:72]: Great use of modifiers!

**Review by aperezsantos — APPROVED** [review]
Awesome! I just have a quick comment about the two .nav classes, which seem to either be a class nested within itself, or two separate classes. If it was done intentional, it's all good. Might just be a lack of understanding on my part.

File: src/assets/styles/scss/details.scss (hunk 1 above)
Comment (TheGuardingDark) [reply src/assets/styles/scss/details.scss:17]: It's something I need to fix within the mix-in file, but it also means going through all the other scss files to correct, so I've been waiting till I finish building all the pages. Nice catch though, thanks!

**Pull request merged**